src/Symfony/Component/Cache/composer.json

```diff
@@ -35,7 +35,7 @@
         "doctrine/cache": "^1.6|^2.0",
         "doctrine/dbal": "^2.7|^3.0",
         "predis/predis": "^1.1",
-        "psr/simple-cache": "^1.0",
+        "psr/simple-cache": "^1.0|^2.0",
```

Comment:
This should be done in the root composer.json as well, shouldn't it?

Reply:
right, updated